doc/hardware/peripherals/sensor.rst

```diff
@@ -29,13 +29,22 @@ description and units of measurement:
 Values
 ======
 
-Sensor devices return results as :c:struct:`sensor_value`.  This
+Sensor stable APIs return results as :c:struct:`sensor_value`.  This
 representation avoids use of floating point values as they may not be
 supported on certain setups.
 
+A newer API is available in parallel that represents data in a Q31 format
```

Comment:
that can interpret raw sensor data

The API passes along the raw data and decoding is optional